### PR TITLE
297 student section location

### LINKED ISSUE #297: add location to student views where needed

### PR DESCRIPTION
**Pull request by oceancypress**
test: check student current schedule and registration for location

Closes #297

### CODE REVIEW

File: student-info-system/professor/tests/tests_views.py [hunk 1]
```diff
@@ -16,9 +16,11 @@ def setUpTestData(self):
                                       first_name='p',
                                       last_name='rof',
                                       password='hello')
+        u2 = User.objects.create_user(username='u2', first_name='s', password='hello')
         profile = Profile.objects.create(user=u1, role='P')
+        sprofile = Profile.objects.create(user=u2, role='S')
         prof = Professor.objects.create(profile=profile, major=major)
-
+        stud = Student.objects.create(profile=sprofile, major=major)
```
Comment (bjmckenz) [inline student-info-system/professor/tests/tests_views.py]: can you use createStudent, etc? that'll isolate the changes

**Review by jeffreicher — COMMENTED** [review]
code review looks good

File: student-info-system/sis/models.py [hunk 2]
```diff
@@ -466,7 +468,7 @@ class SectionStudent(models.Model):
     )
     grade = models.SmallIntegerField(
         choices=GRADES,
-        default=None,
+        default=GRADE_NG,
```
Comment (bjmckenz) [inline student-info-system/sis/models.py]: not persuaded by this. I'd expect default to be None. Can you tell me more?

**Review by bjmckenz — CHANGES_REQUESTED** [review]

**Review by jeffreicher — APPROVED** [review]
checked in the branch - looks good ready to merge @oceancypress

File: student-info-system/professor/tests/tests_views.py (hunk 1 above)
Comment (oceancypress) [reply student-info-system/professor/tests/tests_views.py]: done

File: student-info-system/sis/models.py (hunk 2 above)
Comment (oceancypress) [reply student-info-system/sis/models.py]: fixed

**Pull request merged**